WaterLog_Backend/Controllers/TankMonitorsController.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Cors;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Storage;
+using Microsoft.Extensions.Configuration;
+using WaterLog_Backend.Models;
+
+
+namespace WaterLog_Backend.Controllers
+{
+        [Route("api/[controller]")]
+        [ApiController]
+        public class TankMonitorsController : ControllerBase
+        {
+        private readonly DatabaseContext _db;
+        readonly IConfiguration _config;
+        public TankMonitorsController(DatabaseContext context, IConfiguration config)
+        {
+            _db = context;
+            _config = config;
+        }
+        // GET api/segments
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<TankMonitorsEntry>>> Get()
```

Comment:
What are you getting, tank monitors? maybe rename your method to GetWhateverYouGetting

Reply:
Done

Reply:
😂 bro.....

Reply:
How is it done when its not renamed?